.github/workflows/node.js.yml

```diff
@@ -16,7 +16,8 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        os: [ubuntu-latest, windows-2016]
```

Comment:
When we need to target more OS we can simply add the target version in this matrix.